src/test/groovy/edu/ie3/datamodel/utils/ContainerUtilsTest.groovy

```diff
@@ -293,6 +294,23 @@ class ContainerUtilsTest extends Specification {
 		actual == expected
 	}
 
+	def "The container utils build a valid distance weighted graph model correctly"(){
+		given:
+		JointGridContainer grid = ComplexTopology.grid
+
+		when:
+		Optional<DistanceWeightedGraph> resultingGraphOpt = ContainerUtils.getDistanceTopologyGraph(grid)
+
+		then:
+		resultingGraphOpt.isPresent()
+		DistanceWeightedGraph resultingGraph = resultingGraphOpt.get()
+
+		resultingGraph.vertexSet() == ComplexTopology.grid.getRawGrid().getNodes()
+
+		resultingGraph.edgeSet().size() == 7
```

Comment:
I would also expect a test for `ContainerUtils#addDistanceGraphEdge(DistanceWeightedGraph, NodeInput, NodeInput)`, as this is _the_ crucial point in this concept.